src/ucp/tag/rndv.c

```diff
@@ -1015,7 +1015,7 @@ static ucs_status_t ucp_rndv_pipeline(ucp_request_t *sreq, ucp_rndv_rtr_hdr_t *r
         return UCS_ERR_UNSUPPORTED;
     }
 
-    rndv_size        = rndv_rtr_hdr->size;
+    rndv_size        = ucs_min(rndv_rtr_hdr->size, sreq->send.length);
```

Comment:
shouldn't `sreq->send.length` always be <= `rndv_rtr_hdr->size` due to receive buffer bust be >= send buffer? Otherwise it's truncate error, right?

Reply:
in pipeline rndv case,  RTR request can be for a fraction of receive length,  which can be less than sreq->send.length

Reply:
i see, thanks for clarification!